src/main/java/dk/kb/datahandler/api/v1/impl/DsDatahandlerApiServiceImpl.java

```diff
@@ -156,6 +156,10 @@ public String indexSolr(@NotNull String origin, Long mTimeFrom, IndexTypeDto typ
         try {
             switch (type){
                 case FULL:
+                    if (mTimeFrom != 0 || mTimeFrom >0) {
```

Comment:
Minor thing, but cant this if statement be simplified to:  `if (mTimeFrom != 0)`